data/src/main/java/com/moez/QKSMS/migration/QkRealmMigration.kt

```diff
@@ -253,6 +249,12 @@ class QkRealmMigration @Inject constructor(
             version++
         }
 
+        if (version == 12L) {
+            realm.schema.get("Conversation")
+                ?.addField("draftDate", Long::class.java, FieldAttribute.REQUIRED)
+            version++
+        }
+
```

Comment:
Does this change also improve things for devices that are still based on Android 11?
Because that's the case for me.